testing/regress/ecl/spray_replicate_test.ecl

```diff
@@ -0,0 +1,151 @@
+/*##############################################################################
+
+    Copyright (C) 2018 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//noHThor
+
+//class=spray
+
+import std.system.thorlib;
+import Std.File AS FileServices;
+
+jlib:= SERVICE
+    unsigned8 rtlTick() : library='jlib',eclrtl,entrypoint='rtlNano';
+END;
+
+dropzonePath := '/var/lib/HPCCSystems/mydropzone/' : STORED('dropzonePath');
+espIpPort := 'http://127.0.0.1:8010/FileSpray' : STORED('espIpPort');
+engine := thorlib.platform();
+prefix := '~regress::' + engine + '-';
+suffix := '-' + jlib.rtlTick() : STORED('timeStamp');
```

Comment:
was this to make filenames unique?

If so, using + WORKUNIT is the way other tests do that.